app/src/main/java/com/hammer/talkbbokki/presentation/topics/TopicListScreen.kt

```diff
@@ -191,8 +221,7 @@ fun CardItem(definiteIndex: Int, currentOffset: Float) {
                 // 중간으로 올수록 0도에 가까워짐. (카드 사이 각도 15도씩 틀어짐)
                 lerp(
                     start = pageOffsetWithSign * 15f.dp, // -30, -15, 0, 15, 30 ...
-                    stop = 0f.dp,
-                    fraction = 1f - pageOffset.coerceIn(0f, 1f)
+                    stop = 0f.dp, fraction = 1f - pageOffset.coerceIn(0f, 1f)
```

Comment:
🚫 **[ktlint]** <argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)